tasks/pkg-redhat/install-apm-inject-latest.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Could we mimic what we have in the `tasks/pkg-redhat/install-latest.yml` and have one task for `dnf` and another one for `yum`?

Reply:
changed!